File: app/src/main/java/me/lwoollett/quotely/model/Contents.java

package me.lwoollett.quotely.model;

import java.util.HashMap;
import java.util.List;
import java.util.Map;

public class Contents {

    private List<Quote> quotes = null;
    private String copyright;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    /**
     * No args constructor for use in serialization
     * 
     */
    public Contents() {
    }

    /**
     * 
     * @param quotes
     * @param copyright
     */
    public Contents(List<Quote> quotes, String copyright) {
        super();
        this.quotes = quotes;
        this.copyright = copyright;
    }

    public List<Quote> getQuotes() {
        return quotes;
    }

    public void setQuotes(List<Quote> quotes) {
        this.quotes = quotes;
    }

    public String getCopyright() {
        return copyright;
    }

    public void setCopyright(String copyright) {
        this.copyright = copyright;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/me/lwoollett/quotely/model/Contents.kt
package me.lwoollett.quotely.model

import java.util.HashMap

class Contents {

    var quotes: List<Quote>? = null
    var copyright: String? = null
    val additionalProperties: Map<String, Object> = HashMap<String, Object>()

    /**
     * No args constructor for use in serialization
     *
     */
    constructor() {}

    /**
     *
     * @param quotes
     * @param copyright
     */
    constructor(quotes: List<Quote>, copyright: String) : super() {
        this.quotes = quotes
        this.copyright = copyright
    }

    fun setAdditionalProperty(name: String, value: Object) {
        this.additionalProperties.put(name, value)
    }

}
